src/Services/QueuedJobService.php

```diff
@@ -124,6 +124,15 @@ class QueuedJobService
      */
     private static $worker_ttl = 'PT5M';
 
+    /**
+     * Timeout value in seconds for the Initialising state
+     * if a job is stuck in this state longer than this value it's considered stalled
+     *
+     * @var int
+     * @config
+     */
+    private static $initialising_state_ttl = 120;
```

Comment:
Does it make sense to retain the same format across different TTL settings? See `$worker_ttl`.

```suggestion
    private static $initialising_state_ttl = 'PT2M';
```

Reply:
Good spotting, I'll update the configuration format 👍  thanks